### DISCUSSION TITLE
JBrowse 2 - search for gene

### DISCUSSION THREAD

**Discussion by fuzing** [GMOD/jbrowse-components#2085] [Q&A]
Quick question - I know there was the ability to search for a specific gene within JBrowse 1.   Can I do the same thing with JBrowse 2?

**Comment by garrettjstevens** [comment]
This is something we are actively working to bring to JBrowse 2 and hope to have an initial version ready for users in the near future. In JBrowse 2 you will be able to set up a global text searching index (e.g. in your case perhaps an index based on gene names a gff3 file) as in JBrowse 1, but you'll also be able to have track-specific indexes, and there will be multiple types of indexes supported.

You can see some of work we have done an what is in progress by searching for "text search" in our issues and PRs, e.g. https://github.com/GMOD/jbrowse-components/issues?q=%22text+search%22+.

**Comment by fuzing** [comment]
Garrett -

I greatly appreciate the feedback and will take a look at the text search
functionality

Have a great day
